iceoryx_posh/include/iceoryx_posh/internal/runtime/ipc_interface_base.hpp

```diff
@@ -87,6 +87,7 @@ enum class IpcMessageErrorType : int32_t
     NOTYPE,
     /// A publisher could not be created unique
     NO_UNIQUE_CREATED,
+    SERVICE_DESCRIPTION_IS_FORBIDDEN,
```

Comment:
or `INTERNAL_SERVICE_DESCRIPTION_NOT_ALLOWED` or `ILLEGAL_SERVICE_DESCRIPTION`?

Reply:
Changed to `INTERNAL_SERVICE_DESCRIPTION_IS_FORBIDDEN`.